pkg/operation/botanist/component/logging/eventlogger/logging_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eventlogger_test
+
+import (
+	. "github.com/gardener/gardener/pkg/operation/botanist/component/logging/eventlogger"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Logging", func() {
+	Describe("#CentralLoggingConfiguration", func() {
+		It("should return the expected logging parser and filter", func() {
+			loggingConfig, err := CentralLoggingConfiguration()
+
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(loggingConfig.Filters).To(Equal(`[FILTER]
+    Name                nest
+    Match               kubernetes.*event-logger*event-logger*
+    Operation           lift
+    Nested_under        log
+
+[FILTER]
+    Name                record_modifier
+    Match               kubernetes.*event-logger*event-logger*
+    Record 				job event-logging
+`))
+			Expect(loggingConfig.UserExposed).To(BeTrue())
```

Comment:
I cannot access the event-logger logs from the users grafana. Returns nothing.

Reply:
ok